app/views/reservation_mailer/confirmed.html.erb

```diff
@@ -0,0 +1,54 @@
+<h3>Hi <%= @reservation.customer.name.titleize %>!</h3>
+<p>
+
+  Congrats, Your booking for <%= @reservation.person_count %> guests has been successfully made.
+  <br><br>
+  Here, is the booking details for the same:
+  <table>
+    <tr> 
+      <td>Branch Address</td>
+      <td><%= @reservation.branch.address %></td>
+    </tr>
+    <tr>
+      <td>Booking Date</td>
+      <td><%= @reservation.booking_schedule.to_date %></td>
+    </tr>
+    <tr>
+      <td>Booking Time</td>
+      <td><%= @reservation.booking_schedule.to_s(:time) %></td>
+    </tr>
+  </table>
+
+  <% if @line_items.exists? %>
+    <table>
+      <thead>
+        <tr>
+          <th>S.No.</th>
+          <th>Category</th>
+          <th>Name</th>
+          <th>Quantity</th>
+          <th>Price</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @line_items.each_with_index do |line_item, index| %>
+          <tr>
+            <td><%= index + 1 %></td>
+            <td><%= line_item.product.category %></td>
+            <td><%= line_item.product.name %></td>
+            <td><%= line_item.quantity %></td>
+            <td><%= line_item.product.price * line_item.quantity %></td>
```

Comment:
what if product price changes in the future?

Reply:
For this purpose, i have to add another column for the price at which the product is ordered.

Reply:
Buying_price added to line_items tables.